src/cache.cpp

```diff
@@ -747,9 +747,20 @@ void Cache::do_vacuum()
 	run_sql("VACUUM;");
 }
 
-void Cache::cleanup_cache(std::vector<std::shared_ptr<RssFeed>> feeds,
+int Cache::cleanup_cache(std::vector<std::shared_ptr<RssFeed>> feeds,
 	bool always_clean)
 {
+	int unreachable = 0;
+	std::string list = "(";
+
+	for (const auto& feed : feeds) {
+		std::string name =
+			prepare_query("'%q'", feed->rssurl());
```

Comment:
I believe this can be made `const`, and with `auto` it'd even fit into a single line. (I did notice that this code was simply moved, but, you know, the boyscout rule and all that :)